vllm/executor/ray_gpu_executor.py

```diff
@@ -98,13 +97,22 @@ def _init_workers_ray(self, placement_group: "PlacementGroup",
                 num_gpus=num_gpus,
                 scheduling_strategy=scheduling_strategy,
                 **ray_remote_kwargs,
-            )(RayWorkerVllm).remote(self.model_config.trust_remote_code)
+            )(RayWorkerWrapper).remote(
+                init_cached_hf_modules=self.model_config.trust_remote_code,
+                worker_module_name="vllm.worker.worker",
+                worker_class_name="Worker",
+            )
 
             worker_ip = ray.get(worker.get_node_ip.remote())
             if worker_ip == driver_ip and self.driver_dummy_worker is None:
                 # If the worker is on the same node as the driver, we use it
                 # as the resource holder for the driver process.
                 self.driver_dummy_worker = worker
+                self.driver_worker = RayWorkerWrapper(
+                    init_cached_hf_modules=self.model_config.trust_remote_code,
```

Comment:
nit: if we pass `trust_remote_code` directly, the implementation itself can decide whether it needs to do `init_cached_hf_modules`. otherwise since the decision is not "owned" by the wrapper, it becomes more coupled with higher layer. (I realize this isn't your addition, OK to ignore)

Reply:
fixed in [74deb44](https://github.com/vllm-project/vllm/pull/4024/commits/74deb444c1a694b940720806e8b318da562273f5).